graphql/schema/types/task.graphql

```diff
@@ -111,10 +114,9 @@ type Task {
   spawnHostLink: String
   startTime: Time
   """
-  This is a task's display status and is what is commonly used on the UI.
-  In future releases this will be migrated to represent the original status of the task
+  This is a tasks original status before the displayStatus was added
```

Comment:
nit. would add some kind of comment that makes it clear what this status is (feel free to correct if what I said was wrong)
```suggestion
  This is a task's original status. It is the status stored in the database, and is distinct from the displayStatus.
```